@AH-64D Apache Longbow/Addons/fza_ah64_sfmplus/functions/simpleRotor/fn_simpleRotorTail.sqf

```diff
@@ -0,0 +1,134 @@
+
+//Calculate the hover induced velocity
+private _sign                      = [_rtrThrust] call fza_fnc_sign;
+private _rtrInducedVelocity        = sqrt((abs _rtrThrust) / (2 * _dryAirDensity * _rtrArea)) * _sign;
+//Gather the velocities required to determine the actual induced flow velocity using the newton-raphson method
+private _w = _rtrInducedVelocity;
+private _u = if (_w == 0) then { 0.0; } else { _velYZ / _rtrInducedVelocity; };
+private _n = if (_w == 0) then { 0.0; } else { _velX  / _rtrInducedVelocity; };
+private _rtrCorrInducedVelocity    = if (_w == 0) then { 0.0; } else { [_w, _u, _n] call fza_sfmplus_fnc_simpleRotorNewtRaphSolver; };
+_rtrCorrInducedVelocity            = _rtrCorrInducedVelocity * _rtrInducedVelocity;
+//Calculate the required rotor power
+private _rtrPowerScalar            = [_rtrPowerScalarTable, _altitude] call fza_fnc_linearInterp select 1;
+private _rtrPowerReq               = (_rtrThrust * _velX + _rtrThrust * _rtrCorrInducedVelocity) * _rtrPowerScalar;
+//Calculate the required rotor torque
+private _rtrTorque                 = if (_rtrOmega == 0) then { 0.0; } else { _rtrPowerReq / _rtrOmega; };
+//Calcualte the required engine torque
+private _reqEngTorque              = _rtrTorque / _rtrGearRatio;
+//_heli setVariable ["fza_sfmplus_reqEngTorque", _reqEngTorque];
+
+private _axisX = [1.0, 0.0, 0.0];
+private _axisY = [0.0, 1.0, 0.0];
+private _axisZ = [0.0, 0.0, 1.0];
+
+private _totalThrust = _rtrThrust;
+private _thrustX     = _axisX vectorMultiply ((_totalThrust * -1.0) * _deltaTime);
+private _torqueY     = ((_rtrTorque  * -1.0) * _rtrTorqueScalar) * _deltaTime;
+private _torqueZ     = ((_rtrPos # 1) * _totalThrust * -1.0) * _deltaTime; 
+
+//Rotor thrust force
+_heli addForce [_heli vectorModelToWorld _thrustX, _rtrPos];
+//Tail rotor torque effect
+_heli addTorque (_heli vectorModelToWorld [0.0, _torqueY, _torqueZ]);
+
+#ifdef __A3_DEBUG__
+[_heli, _rtrPos, _rtrPos vectorAdd _axisX, "red"]   call fza_fnc_debugDrawLine;
+[_heli, _rtrPos, _rtrPos vectorAdd _axisY, "green"] call fza_fnc_debugDrawLine;
+[_heli, _rtrPos, _rtrPos vectorAdd _axisZ, "blue"]  call fza_fnc_debugDrawLine;
+[_heli, 24, _rtrPos, _bladeRadius, 0, "white", 0]   call fza_fnc_debugDrawCircle;
+#endif
+
+/*
+hintsilent format ["v0.7 testing
+                    \nRotor Omega = %1
+                    \nBlade Tip Vel = %2
+                    \nRotor Power Req = %3 kW
+                    \nRotor Torque = %4 Nm 
+                    \nE1 Tq = %5 % E2 Tq = %6 %
+                    \nVelZ = %7
+                    \nInduced Vel Scalar = %8
+                    \nGnd Eff Scalar = %9
+                    \nStab = %10
+                    \nPitch = %11", _rtrOmega, _bladeTipVel, _rtrPowerReq * 0.001, _reqEngTorque, (_reqEngTorque / 2) / 481, (_reqEngTorque / 2) / 481, _velZ, _inducedVelocityScalar, _gndEffScalar, fza_sfmplus_collectiveOutput, _heli call BIS_fnc_getPitchBank select 0];
+                    */
```

Comment:
We should look at ways to share a lot of the info between simpleRotorMain and simpleRotorTail - though leaving for now

Reply:
About the only consistent thing are rotor parameters, there are some pretty significant differences with regards to implementation however